WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -44,6 +44,21 @@ def scramble_program
   def help
   end
 
+  def users_delegates_search
+    users_search(delegate_only=true)
```

Comment:
That's Python's way of doing keyword arguments. In Ruby this creates a local variable called `delegate_only`.

It probably should be

```ruby
def users_search(delegate_only: false)

# and

users_search(delegate_only: true)
```